client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -125,7 +125,7 @@ export const getTask = (
 			break;
 		case CHECKLIST_KNOWN_TASKS.MOBILE_APP_INSTALLED:
 			taskData = {
-				title: translate( 'Try the Jetpack app' ),
+				title: translate( 'Get the mobile app' ),
```

Comment:
It looks good, but I suggest you add a fallback to make non-English users keep seeing the old copy until the translation is done.

Reply:
Good point!
I added the "Status string Freeze" label and will wait till the string is fully translated before merging it.

Reply:
That's good ✅